activesupport/lib/active_support/test_case.rb

```diff
@@ -76,6 +76,7 @@ def parallelize(workers: :number_of_processors, with: :processes)
         workers = ENV["PARALLEL_WORKERS"].to_i if ENV["PARALLEL_WORKERS"]
 
         return if workers <= 1
+        return if ActiveSupport.test_parallelization_disabled && !ENV["PARALLEL_WORKERS"]
```

Comment:
I think it might make be cleaner if `test_parallelization_disabled` was a full method that also did the check for ENV. What do you think?

Reply:
That makes sense, should this method be on `ActiveSupport` or here in `ActiveSupport::TestCase`?

Reply:
I changed it so that the env var presence is checked at the moment of attempting to toggle the flag, it looks cleaner.